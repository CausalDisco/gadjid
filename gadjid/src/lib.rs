// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
