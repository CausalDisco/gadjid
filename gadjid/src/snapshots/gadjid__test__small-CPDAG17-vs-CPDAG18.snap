---
source: src/lib.rs
expression: "test(g_true, g_guess)"
---
g_true: 10-node-CPDAG-17
g_guess: 10-node-CPDAG-18
ancestor_aid:
  - 0
  - 0
oset_aid:
  - 0
  - 0
parent_aid:
  - 0
  - 0
shd:
  - 0.8
  - 36
t:
  - 6
y: 1
z:
  - 0
  - 2
  - 3
  - 4
  - 5
  - 7
  - 8
  - 9
possible_descendants_of_t_in_g_guess:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
not_amenable_in_g_guess_wrt_t:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 7
  - 8
  - 9
proper_ancestors_of_y_in_g_guess_wrt_t:
  - 1
oset_for_t_onto_y_in_g_guess: []
not_validly_adjusted_for_in_g_guess_by_parents_of_t:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 7
  - 8
  - 9
not_validly_adjusted_for_in_g_guess_by_oset_for_t_onto_y:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 7
  - 8
  - 9
not_validly_adjusted_for_in_g_guess_by_empty_set:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 7
  - 8
  - 9
not_validly_adjusted_for_in_g_guess_by_z:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 7
  - 8
  - 9
